pkg/cli/dev.go

```diff
@@ -64,6 +66,11 @@ func (s *Dev) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	// Ensure opts.AutoUpgrade is set if is implied by opts.AutoUpgradeInterval or imageSource.Image's pattern.
+	if opts.AutoUpgrade == nil || !*opts.AutoUpgrade {
+		opts.AutoUpgrade = z.P(autoupgrade.ImpliedAutoUpgrade(opts.AutoUpgradeInterval, imageSource.Image))
+	}
+
```

Comment:
If I understand correctly, this is no longer needed because the UI should be looking at the `status` field. Is that right?

If that is correct, can we remove the other place where this was done previously?

Reply:
Yeah I was on the fence before you mentioned it about this. After some more though (and the current implementation) I think what you're saying makes sense.